### ISSUE TITLE
Add ability to extract audio to two formats

### ISSUE THREAD

**Issue by frankhommers** [jmbannon/ytdl-sub#564]
Hi there!
Great tool!

I would like the ability to extract downloads to multiple audio formats.
My watch loves mp3's.
My Hiby music player loves AAC's.

Is this functionality you would consider?

**Comment by jmbannon** [comment]
Yes actually. Another use case is splitting audio and video. Audio + audio should be supported as well

**Comment by bpoulliot** [comment]
Just spitballing, I may be able to test later, but this might be currently possible with a workflow of using audio_extract + post-processing with ffmpeg as long as you use ytdl_opts to keep the original file. Again, would need to test this as I was at the end of a very long day thinking about this yesterday and not in a place to set up a test case. But may be easy to set up a workaround for now as well as implement on the back end eventually if that's the route you want to go or don't have another idea.

**Comment by jmbannon** [comment]
> Just spitballing, I may be able to test later, but this might be currently possible with a workflow of using audio_extract + post-processing with ffmpeg as long as you use ytdl_opts to keep the original file. Again, would need to test this as I was at the end of a very long day thinking about this yesterday and not in a place to set up a test case. But may be easy to set up a workaround for now as well as implement on the back end eventually if that's the route you want to go or don't have another idea.

Not possible at the moment - the plugin deletes the old file :-)
This is something that should be toggle-able

**Comment by bpoulliot** [comment]
Cool. I won't give it a shot unless this is on the back burner. Figured it would be a quick trial on my end since I know with the original program you're able to keep the original files. Again, might be able to craft a workaround but I'm not interested in throwing wrenches into gears or reinventing wheels that aren't even mine. ;-)

**Comment by jmbannon** [comment]
Having a custom ffmpeg command do this should be straight forward. However, the one thing that still has me thinking about this feature is, how to handle things that come afterward? Such as:
- Moving the file to a separate directory
- Renaming the output file with source/override variables
- Including thumbnails in both directories

The dead-simple approach is to just throw it in the same folder with the same name, but different extension. But I don't think that will be enough for the use-cases that want to use this feature
